packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -104,6 +104,31 @@ export class ICXOrderBook {
     )
   }
 
+  /**
+   * Create and submits a external(EXT) HTLC transaction
+   *
+   * @param {ExtHTLC} htlc
+   * @param {string} htlc.offerTx Transaction Id of the offer transaction for which the HTLC is
+   * @param {BigNumber} htlc.amount Amount in HTLC
+   * @param {string} htlc.htlcScriptAddress Script address of external HTLC
+   * @param {string} htlc.hash Hash of seed used for the hash lock part
+   * @param {string} htlc.ownerPubkey Pubkey of the owner to which the funds are refunded if HTLC timeouts
+   * @param {number} htlc.timeout Timeout (absolute in blocks) for expiration of HTLC in DFI blocks
+   * @param {UTXO[]} [utxos = []] Specific utxos to spend
```

Comment:
```
   * @param {UTXO[]} [utxos = []]
   * @param {string} [utxos.txid]
   * @param {number} [utxos.vout]
```

Apply to your other icx functions too

Reply:
will update it to 
```
* @param {UTXO[]} [utxos = []] Specific utxos to spend
* @param {string} utxos.txid transaction Id
* @param {number} utxos.vout The output number
```

Here `txid` and `vout` is not optional within  `UTXO`